requirements.in

```diff
@@ -0,0 +1,24 @@
+python-dateutil     # date type
```

Comment:
What's this? :)

Reply:
Ah, that's moved over from here as part of pinning:

https://github.com/saulpw/visidata/pull/563/files#diff-b4ef698db8ca845e5845c4618278f29aL1

That whole commit can be left out really, it's too opinionated for this PR. I just figured I was in over my head messing with this stuff and didn't want the extra risk of dependency version changes causing regressions. And it does invite discussion! :)